docs/tutorials/visualizing_the_system.ipynb

```diff
@@ -0,0 +1,3760 @@
+       "  var render_items = [{\"docid\":\"155d690f-7819-41ff-a29f-cb0ea329f6d2\",\"root_ids\":[\"1004\"],\"roots\":{\"1004\":\"06544398-859d-4a08-80f4-d02993640b99\"}}];\n",
+       "  root.Bokeh.embed.embed_items_notebook(docs_json, render_items);\n",
+       "\n",
+       "  }\n",
+       "  if (root.Bokeh !== undefined) {\n",
+       "    embed_document(root);\n",
+       "  } else {\n",
+       "    var attempts = 0;\n",
+       "    var timer = setInterval(function(root) {\n",
+       "      if (root.Bokeh !== undefined) {\n",
+       "        clearInterval(timer);\n",
+       "        embed_document(root);\n",
+       "      } else {\n",
+       "        attempts++;\n",
+       "        if (attempts > 100) {\n",
+       "          clearInterval(timer);\n",
+       "          console.log(\"Bokeh: ERROR: Unable to run BokehJS code because BokehJS library is missing\");\n",
+       "        }\n",
+       "      }\n",
+       "    }, 10, root)\n",
+       "  }\n",
+       "})(window);"
+      ],
+      "application/vnd.bokehjs_exec.v0+json": ""
+     },
+     "metadata": {
+      "application/vnd.bokehjs_exec.v0+json": {
+       "id": "1004"
+      }
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "p = plot_dag(\n",
+    "    functions=policy_functions,\n",
+    "    selectors=[{\"node\": \"kindergeld_m\", \"type\": \"neighbors\"}],\n",
+    ");"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The general interface of the plotting function is similar to ``compute_taxes_and_transfers()``, but without the ``data`` and ``params`` argument.\n",
+    "\n",
+    "## Labels\n",
+    "\n",
+    "It is possible to hide the labels of the nodes by setting `plot_dag(..., labels=False)`.\n",
```

Comment:
Comes a bit out of the blue. Maybe show the entire signature upfront? Or repeat the plot, but might be overkill.

Reply:
Done. Small intro and signature.